packages/eslint-config/library.js

```diff
@@ -4,8 +4,14 @@ const project = resolve(process.cwd(), "tsconfig.json");
 
 /** @type {import("eslint").Linter.Config} */
 module.exports = {
-  extends: ["eslint:recommended", "prettier", "eslint-config-turbo"],
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier",
+    "eslint-config-turbo",
```

Comment:
Is it fair to say `eslint-config-turbo` is relevant for every package in javascript repo?

Asking because we want to share this `eslint-config` package in control dashboard, and turborepo lint rules aren't applicable there. But when I run eslint with this config, only warnings we are getting is "turbo/no-undeclared-env-vars", so maybe it's easier to just keep this in and then turn it off there.

Reply:
Yeah, I think we should keep `eslint-config-turbo` here especially because it can point out ways we're not using turborepo correctly. If it's easy to override the config from the dashboard, let's do that. Otherwise I can take a look at not including it when we publish or have a separate config for `monorepo-library` (or something along those lines)